Apps/AdminWebClient/src/AdminWebClient.csproj

```diff
@@ -72,7 +72,7 @@
     <!-- Include the newly-built files in the publish output -->
     <ItemGroup>
       <DistFiles Include="$(SpaRoot)dist\**" />
-      <ResolvedFileToPublish Include="@(DistFiles->'%(FullPath)')" Exclude="@(ResolvedFileToPublish)">
```

Comment:
just curious, did the > break the build?

Reply:
I'm running on a mac and this gets replaced all the time, I wonder if there is any impact for windows?

Reply:
There is no impact it runs fine on both, I've had the same issues and now leave the escaped version.